dask/dataframe/core.py

```diff
@@ -1755,6 +1756,22 @@ def astype(self, dtype):
         empty = self._pd.astype(dtype)
         return map_partitions(pd.DataFrame.astype, empty, self, dtype=dtype)
 
+    def info(self):
+        """
+        Concise summary of a Dask DataFrame.
+        """
+        from pandas.formats.format import _put_lines
+        lines = list()
+        lines.append(str(type(self)))
+        lines.append('Data columns (total %d columns):' % len(self.columns))
+        dtypes = self.dtypes
+        template = "%s    %s"
+        for i, col in enumerate(self.columns):
+            dtype = dtypes.iloc[i]
+            lines.append(template % (col, dtype))
+
+        sys.stdout.write('\n'.join(lines))
```

Comment:
`capsys` should work either way, as both send to `stdout`. The only difference between these two is that `print` appends a newline to the end of the output, so you'd have to add a `'\n'` at the end of your test string. I think you do want the newline at the end though, otherwise it looks odd in the repl:

```python
In [13]: sys.stdout.write('foo')
foo
In [14]: sys.stdout.write('foo\n')
foo

In [15]: print('foo', end='')
foo
In [16]: print('foo')
foo
```

Not a big deal, up to you.